classes/class-connector.php

```diff
@@ -91,10 +91,6 @@ public function register() {
 	 * Unregister all context hooks
 	 */
 	public function unregister() {
-		if ( ! $this->is_registered ) {
```

Comment:
@dero Because I noticed that somewhere between tests the connectors where ending in a state where the hooks were `is_registered` flag was `false` but the action were still registered and somehow prevent the mocked connector class actions from getting the data on logs catching deletions. This has actually been happening across multiple connector test and is also the cause for hold up on the last test of the comment connector test case